src/corecel/sys/ScopedMem.cc

```diff
@@ -116,7 +118,7 @@ ScopedMem::ScopedMem(std::string_view label, MemRegistry* registry)
 /*!
  * Register data on destruction.
  */
-ScopedMem::~ScopedMem()
+ScopedMem::~ScopedMem()  // NOLINT(bugprone-exception-escape)
```

Comment:
Does this just need a `noexcept`?

Reply:
`clang-tidy-15` doesn't respect the `noexcept(false)`, this work on my local build.